onyo/commands/new.py

```diff
@@ -67,6 +89,12 @@ def create_faux(onyo_root, list_of_assets, faux_length=8):
 
 
 def create_filename(onyo_root, template):
```

Comment:
That is true. 
There was already a discussion about using e.g. the git index to get a list of files and changing the function. I will for  not update this now, instead I created an issue for this and work on it in another PR for the refactoring of some of `util`s functions.

Reply:
#218 describes this.

Reply:
At bare minimum, this can be trivially reduced from 5x to 1x.

The full problem does not need to be solved immediately in order to drastically reduce the amount of work done here.

Reply:
I was not exactly sure how a trivial partial solution would look like, instead I decided to do this with the git index.

The newest commit is a refactoring of the function and it would be nice if you could review the changes in `get_list_of_assets()` that I made there.